exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var message = "This is a string."
+console.log(message)
+console.log(typeof(message));
```

Comment:
You don't need brackets after `typeof`, it is simply
```js
typeof message
```

Reply:
> You don't need brackets after `typeof`, it is simply
> 
> ```js
> typeof message
> ```

Thank you!